### PR TITLE
RavenDB-6069 Documents with nullable properties

### PR DESCRIPTION
**Pull request by efratshenhar**

### CODE REVIEW

File: src/Raven.NewClient/Json/BlittableJsonReader.cs [hunk 1]
```diff
@@ -183,6 +183,9 @@ private bool SetToken(BlittableJsonToken token, object value)
             if (TokenType == JsonToken.EndArray)
                 return new int?();
 
+            if (TokenType == JsonToken.Null)
+                return new int?();
```
Comment (ayende) [inline src/Raven.NewClient/Json/BlittableJsonReader.cs:187]: What about nullable float? nullable bool, etc?

**Review by ayende — CHANGES_REQUESTED** [review]

File: src/Raven.NewClient/Json/BlittableJsonReader.cs (hunk 1 above)
Comment (efratshenhar) [reply src/Raven.NewClient/Json/BlittableJsonReader.cs:187]: float, bool ,double already working

Comment (ayende) [reply src/Raven.NewClient/Json/BlittableJsonReader.cs:187]: Do we have tests for them?

Comment (efratshenhar) [reply src/Raven.NewClient/Json/BlittableJsonReader.cs:187]: yes for bool. I will add for float

**Pull request merged**